src/main/java/frc/robot/constants/GameConstants.java

```diff
@@ -11,11 +11,11 @@ public class GameConstants {
     public static final int DIAG_ABS_SPARK_ENCODER = 20;
 
     //DEBUGS
-    public static final boolean SHOOTER_DEBUG = false;
-    public static final boolean RAMP_DEBUG = false;
-    public static final boolean SWERVE_DEBUG = false;
-    public static final boolean FEEDER_DEBUG = false;
+    public static final boolean SHOOTER_DEBUG = true;
```

Comment:
Now that you've confirmed logging is working, you should not flip these to true in your PR.